src/main/java/com/google/sps/servlets/FreeTimeServlet.java

```diff
@@ -125,9 +119,11 @@ public void doGet(
       freeTimeUtility.addEvent(eventStart, eventEnd);
     }
 
-    int wordCount = getWordCount(googleCredential);
+    int numberDaysUnread = 7;
+    int wordCount = gmailClient.getWordCount(numberDaysUnread);
     int averageReadingSpeed = 50;
-    int minutesToRead = (int) Math.ceil((double) wordCount / averageReadingSpeed);
+    int minutesToRead;
+    minutesToRead = wordCount == 0 ? 0 : (int) Math.ceil((double) wordCount / averageReadingSpeed);
```

Comment:
nit: simplify to `(int) Math.ceil((double) wordCount / averageReadingSpeed);`

Reply:
Agreed. I was trying to account for when averageReadingSpeed would be 0, but that should never be the case to begin with. This was redundant. `Resolved`

Reply:
Out of curiosity, to avoid manipulating floating point numbers, you can use the follow identity:
`ceil(a/b) = floor((a+b-1)/b)` (and the floor is what an integer division will give you by default in most languages.)